src/meshes/t8code_mesh.jl

```diff
@@ -12,6 +12,8 @@ mutable struct T8codeMesh{NDIMS, RealT <: Real, IsParallel, NDIMSP2, NNODES} <:
     forest      :: Ptr{t8_forest} # cpointer to forest
     is_parallel :: IsParallel
 
+    unsaved_changes::Bool
```

Comment:
Where is this required?

Reply:
It is set e.g. in the AMR callback.

Reply:
It is actually never called for `T8codeMesh`es. I deleted this member variable. All in the sense of YAGNI.

Reply:
If it is never used, does that mean that when the `SaveSolutionCallback` stores the solution, the corresponding modified mesh is not written out as well? Or is it never used in the sense that the SSC will _always_ write out the current T8code mesh, even if it has not changed?

Reply:
`SaveSolutionCallback` ist not yet implemented for `T8codeMesh`.